docs/developer-docs/daos/sns/launching/launch-steps-1proposal.mdx

```diff
@@ -91,11 +91,24 @@ Nothing technical for dapp developers to do. Community votes.
 Nothing technical for dapp developers to do. This is triggered automatically as a result
 of an adopted proposal in Stage 4.
 
-### 6. (Automated) SNS-W sets SNS root as sole controller of dapp.
+### 6. (Automated) SNS-W sets SNS root and NNS root as sole controllers of dapp.
 
 Nothing technical for dapp developers to do. This is triggered automatically as a result
 of an adopted proposal in Stage 4.
 
+:::warning
+At this point, the SNS is in a restricted mode and some SNS proposal types cannot be submitted:
+
+1. Manage Nervous System Parameters
```

Comment:
Could we replace these with the actual proposal names that are restricted?

Reply:
Done!